plugins/flutter_aepedge/pubspec.yaml

```diff
@@ -13,6 +13,7 @@ dependencies:
   flutter:
     sdk: flutter
   flutter_aepcore: ">= 1.0.0 <2.0.0"
+  #TODO
   #flutter_aepedgeidentity: ">= 1.0.0 <2.0.0"
```

Comment:
do we plan to release edgeidentity first, then enable this and release edge?

Reply:
Chris said we can release EdgeIdentity first and then uncomment it for Edge. We can keep this uncommented right now.